ci/jenkins_build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -euo pipefail
+
+# Test configuration environment:
+# KUBE_NODE_TAG:   Docker tag for the Kubernetes node image. This
+#                  determines the Kubernetes version of the KinD cluster.
+#                  Available versions can be found here:
+#                      https://github.com/kubernetes-sigs/kind/releases
+# KUBECTL_VERSION: Version of kubectl to build into test client.
+# CLUSTER_ID:      Cluster identifier string. This will be combined with the
+#                  GIT_REF to generate a unique test client image tag and
+#                  a unique KinD cluster name for this test run.
+# GIT_REF:         Git reference for Conjur OSS helm chart branch being
+#                  tested.
+export KUBE_NODE_TAG=${KUBE_NODE_TAG:-v1.17.5}
+KUBECTL_VERSION=${KUBECTL_VERSION:-1.17.5}
+CLUSTER_ID=${CLUSTER_ID:-1.17.5}
+GIT_REF=${GIT_REF:-master}
+
+echo "==================================="
+echo "Test Configuration:"
+echo "KUBE_NODE_TAG:   $KUBE_NODE_TAG"
+echo "KUBECTL_VERSION: $KUBECTL_VERSION"
+echo "CLUSTER_ID:      $CLUSTER_ID"
+echo "GIT_REF:         $GIT_REF"
+echo "==================================="
+
+KIND_VERSION=0.7.0
+export CLUSTER_NAME="${CLUSTER_ID}-${GIT_REF}"
+TEST_CLIENT_IMAGE="conjur-test-client:${CLUSTER_NAME}"
+
+CURRENT_DIR=$("$(dirname "$0")/abspath.sh")
+TOPLEVEL_DIR="${CURRENT_DIR}/.."
+
+# Build the test client image. This will contain:
+#    - KinD binary
+#    - kubectl binary
+#    - Helm client
+#    - Conjur Helm charts
+echo "==================================="
+echo "Building test client image..."
+echo "==================================="
+docker build -t "${TEST_CLIENT_IMAGE}" \
+             -f ${CURRENT_DIR}/Dockerfile \
+             --build-arg KIND_VERSION="${KIND_VERSION}" \
+             --build-arg KUBECTL_VERSION="${KUBECTL_VERSION}" \
+             "${TOPLEVEL_DIR}"
```

Comment:
Since this is just a test but a long and parallel one, I wonder if we can try using `DOCKER_BUILDKIT=1` for faster runs.

Reply:
Ooh, yeah, this is cool. I like the new build output format too.